lib/github_service/commands/cross_repo_test.rb

```diff
@@ -75,6 +75,9 @@ class CrossRepoTest < Base
 
       restrict_to :organization
 
+      # Cache the repo groups yaml file from https://github.com/ManageIQ/manageiq-release
+      cache_with_timeout(:repo_groups_hash, 60.minutes) { fetch_manageiq_release_repo_groups }
```

Comment:
I think it might be cleaner if the fetch method one-time expands the known list and just stores that as a hash.  Then later, it's a straight hash lookup, with not reiteration.

Reply:
You mean don't cache_with_timeout, just fetch once and cache it?

Reply:
I think I misread the code, because I saw the expansion in a loop and though it was doing some more complicated expansion the list on each pass.